CHANGELOG.md

```diff
@@ -5,42 +5,85 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Released]
+## [Release]
+
+### [0.1.8] - 2018-11-15
+
+#### Added
+
+- Support for Union and TypeCondition filtering
+
+#### Changed
+
+Now a resolver of a union type should explicitly define the value of `_typename` on it's return value.
+
+- If the resolver wants to return a `dict` a `_typename` key should be present.
+- If the resolver wants to return an `object` (a class instance), a `_typename` field should be present.
```

Comment:
I wondering if it is called a `field` or `attribute`? What do you think?

`should` -> `must`

Reply:
Yeah good catch, it's class attribute, I was thinking about graphql field. :)

Reply:
but it stay should as if the class is correctly named, it will work.